### PR TITLE
Adding step to set tz to UTC

### PR DESCRIPTION
**Pull request by Semantic-spaces**

### CODE REVIEW

File: roles/spads/tasks/initialSetup.yaml [hunk 1]
```diff
@@ -12,6 +12,19 @@
     name: "*"
     state: latest
 
+- name: Copy using inline content
+  ansible.builtin.copy:
+    content: 'Etc/UTC\n'
+    dest: /etc/timezone
+    owner: root
+    group: root
+    mode: '0644'
```
Comment (p2004a) [inline roles/spads/tasks/initialSetup.yaml:21]: I don't believe we need to set the owner, group and mode: this is already executed as root, so no need to set explicitly.

**Review by p2004a — COMMENTED** [review]
I assume you tested those commands on some vm or something?

I've skimmed through https://wiki.debian.org/TimeZoneChanges but it's a bit confusing on what's exactly the proper procedure, maybe you should just use https://docs.ansible.com/ansible/latest/collections/community/general/timezone_module.html and do it in 2 lines (we already depend on `community.general`).

**Comment by Semantic-spaces** [comment]
Updated as per comments

**Comment by Semantic-spaces** [comment]
> I assume you tested those commands on some vm or something?
> 
> I've skimmed through https://wiki.debian.org/TimeZoneChanges but it's a bit confusing on what's exactly the proper procedure, maybe you should just use https://docs.ansible.com/ansible/latest/collections/community/general/timezone_module.html and do it in 2 lines (we already depend on `community.general`).

If we're already using community.general than I agree it makes sense, I updated the PR.

**Review by p2004a — APPROVED** [review]
LGTM, I will push it to servers and then merge.

**Comment by p2004a** [comment]
I made a mistake when rebasing, pushed in https://github.com/beyond-all-reason/ansible-spads-setup/pull/31

**Pull request closed without merging**